src/components/Tooltip/Tooltip.spec.tsx

```diff
@@ -2,22 +2,113 @@
 
 import { mount } from "@cypress/react";
 import React from "react";
-import { Tooltip } from "./Tooltip";
+import { Tooltip, TooltipButton } from "./Tooltip";
+import IconIcons from "@elements/Icon/Generated/IconIcons";
+import { BrightHeaderStyle } from "./BrightHeader";
 
 const TOOLTIP_TEXT = "This is a tooltip";
 const TOOLTIP_ICON_ID = "[data-test-id=tooltip-icon]";
 const TOOLTIP_ID = "[data-test-id=tooltip]";
+const TOOLTIP_LINK_ID = "[data-test-id=tooltip-link]";
+const TOOLTIP_LINK_URL = "http://www.frontify.com";
```

Comment:
```suggestion
const TOOLTIP_LINK_URL = "https://www.frontify.com";
```
To fix SonarCloud issue